media-gfx/hydrus/hydrus-442.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_REQ_USE="sqlite"
+
+inherit python-single-r1 desktop optfeature
+
+DESCRIPTION="A booru-like media organizer for the desktop"
+HOMEPAGE="http://hydrusnetwork.github.io/hydrus/ https://github.com/hydrusnetwork/hydrus"
```

Comment:
https

Reply:
Fixed this one, for some reason it seems that QtPy/pyside2 didn't pull the python 3.9 version for you? I'm pretty sure I put the usedeps on both, that's strange.